homeassistant/components/fastdotcom/__init__.py

```diff
@@ -57,12 +62,12 @@ def update(call=None):
 class SpeedtestData:
     """Get the latest data from fast.com."""
 
-    def __init__(self, hass):
+    def __init__(self, hass: HomeAssistant) -> None:
         """Initialize the data object."""
-        self.data = None
+        self.data: dict[str, Any] | None = None
         self._hass = hass
 
-    def update(self, now=None):
+    def update(self, now: datetime | None = None) -> None:
```

Comment:
it looks like the `now` parameter is not used anywhere? do I miss something?

Reply:
Removed and tested, didn't break anything fortunatly